students/Shweta/Lesson09/donor_model.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+
+'''thankyou message for donation'''
+mty_msg="\n".join(("Thankyou {}",
+                   "for your generous donation of {:.2f}",))
+
+
+ltr="\n".join(("", "Dear {},",
+              "Thank you for your very kind donation of ${:.2f}.",
+              "It will be put to very good use.",
+              "           Sincerely,",
+              "               -The Team"))
+
+class Donor:
+
+    def __init__(self,name,tamt):
+        self.name = name
+        self.tamt = tamt
+
+    def add_donationamt(self,damt):
+        self.tamt.append(damt)
+
+    @property
+    def tn_times(self):
+        return len(self.tamt)
+    
+    @property 
+    def total_sum(self):
+        return sum(self.tamt)
+
+    @property
+    def avg(self):
+        return (self.total_sum/self.tn_times)
+
+
+    def ind_letter(self):
+        indv_letter=ltr.format(self.name,self.total_sum)
+        return indv_letter
+    
+    def formatting(self):
+        self.frmt='{:20} ${:>20.2f}   {:18}   ${:20.2f}'.format(self.name,self.total_sum,self.tn_times,self.avg)
+        return self.frmt
```

Comment:
instance attributes should be defined in `__init__` - however I don't think you need this to be an instance attrb.

this just be `frmt = ...` or 
`return '{:20} ${:>20.2f}   {:18}   ${:20.2f}'.format(self.name,self.total_sum,self.tn_times,self.avg)`